ospfd/ospf_vty.c

```diff
@@ -6416,7 +6416,7 @@ static int show_network_lsa_detail(struct vty *vty, struct ospf_lsa *lsa,
 	}
 
 	if (json)
-		json_object_object_add(json, "attchedRouters",
+		json_object_object_add(json, "attachedRouters",
```

Comment:
While technically correct, anyone already using the json output of this command and is interested in the attached Routers has just been broken.  I would suggest leaving this one alone, adding a new one and deprecating the old value so people have a chance to move to the 'correct' json value

Reply:
I think we really need to come up with a consistent policy here and follow it.